PullToRefresh/PullToRefresh.swift

```diff
@@ -47,7 +47,7 @@ public class PullToRefresh: NSObject {
     
     // MARK: - State
     
-    var state: State = .Initial {
+    public private(set) var state: State = .Initial {
```

Comment:
Now I might be wrong, however, if it is then set to `internal`, and I use it in a Swift module, then even if I subclass it I won't be able to touch it. If it should be possible to subclass and change the state then perhaps it should just be `public`, since there is no `protected` modifier in Swift (afaik)